psi4/driver/driver_findif.py

```diff
@@ -25,20 +25,129 @@
 #
 # @END LICENSE
 #
+"""Plan, run, and assemble QC tasks to obtain derivatives by finite difference of lesser derivatives.
```

Comment:
What is this style of docstring? I'm not familiar with it and find it difficult to read. What is trying to accomplish?

Reply:
There's no particular style to the module-level docstring. It's trying to convey what the wrapper does in a start-to-finish manner that was lost with the class conversion. All bullets at an indentation level happen in the function name of that indentation level. An indent means a called fn that could be either in-class or free.

Reply:
So you're saying this is supposed to show a linear path through the class?

Reply:
yes, from initializing the class (happens in task_planner.py) through to returning the wfn (happens when the driver,py fns call get_psi_results().